### ISSUE TITLE
Sure hub needed?

### ISSUE THREAD

**Issue by tispokes** [benleb/surepetcare#62]
Hi, is the original Hub needed or how do I connect the flap ro HA?

**Comment by 0rsted** [comment]
Yes, the hub is needed, at least until someone decodes the transmissions, and makes an open source hub.

**Comment by netphantm** [comment]
can we change this into a feature request? :)
I have a pretty unstable internet connection and it's a PITA to lock/unlock the flap through the bloody surepet cloud..
